extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -238,12 +239,25 @@ export class ConsentUI {
    * @return {!Element}
    */
   createPlaceholder_() {
-    // TODO(@zhouyx): Allow publishers to provide placeholder upon request
     const placeholder = this.parent_.ownerDocument.createElement('placeholder');
     toggle(placeholder, false);
     const {classList} = placeholder;
-    classList.add(consentUiClasses.fill);
-    classList.add(consentUiClasses.placeholder);
+    classList.add(consentUiClasses.loadingPlaceholder);
+
+    const loadingSpinner = htmlFor(placeholder)`
```

Comment:
nit: Is it possible to declare the string as a const at the top of the file?

Reply:
I believe @kristoferbaxter was looking into doing a compilation pass to do this, so it may not be worth worrying about.

Reply:
Yeah I originally had it as a const at the top of the file, but from what @kristoferbaxter showed me, it appeared to be using the `htmlFor` keyword in order to find the template strings to then be optimized. Thus I did it this way to help with that.

Though I may be wrong, we will see what Kris says 😄

Reply:
Since this `htmlFor` usage is within a method, it will be hoisted automatically in the single pass build. Makes the most sense within source code to place it where it is easiest to understand.

Please note: This optimization is only true in the single-pass build.

Reply:
Awesome! Thanks!

So I tried hoisting this to the top of the file, and the linter then throws syntax errors. Also, tried binding it with `${placeholderHtml}`, but the template can't take in variables. I think we may need to do some additional work to allow the codebase to handle this.

Doing a quick project search, seems `htmlFor()` always has the template immediately after it.